build/build-info

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 #
-# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
This needs to be 2019-2020.  We don't give up copyright for older years.

Reply:
> This needs to be 2019-2020. We don't give up copyright for older years.

Ok